### PR TITLE
Added Image Cropper to the Upload Picture in Users and Courses  - Refs #7683

### PR DESCRIPTION
**Pull request by jloguercio**

### CODE REVIEW

File: main/inc/lib/usermanager.lib.php [hunk 1]
```diff
@@ -3057,6 +3062,34 @@ public static function resize_picture($file, $max_size_for_picture)
...
     }
+    
+    /**
+     * Crop a Image
+     * 
+     * @author José Loguercio <jose.loguercio@beeznest.com>
+     * @param string file picture
+     * @param string crop Parameters (x, y, width, height)
+     * @return obj image object
+     */
```
Comment (jmontoyaa) [inline main/inc/lib/usermanager.lib.php]: Function should be cropImage()

File: main/inc/lib/course.lib.php [hunk 2]
```diff
@@ -5375,6 +5378,67 @@ public static function separateUsersGroups($to)
...
+    
+    /**
+     * Resize a picture
+     *
+     * @param  string file picture
+     * @param  int size in pixels
+     * @todo move this function somewhere else image.lib?
+     * @return obj image object
+     */
+    public static function resize_picture($file, $max_size_for_picture)
```
Comment (jmontoyaa) [inline main/inc/lib/course.lib.php]: resizePicture

File: main/course_info/infocours.php [hunk 3]
```diff
@@ -427,7 +455,18 @@ function is_settings_editable()
...
-
+    
+    // update course picture
+    $picture = $_FILES['picture'];
+    if (!empty($picture['name'])) {
+        $picture_uri = CourseManager::update_course_picture(
+            $course_code,
+            $picture['name'],
+            $picture['tmp_name'],
+            $updateValues['croppResult']
```
Comment (jmontoyaa) [inline main/course_info/infocours.php]: cropResult en lugar de croppResult

**Comment by ywarnier** [comment]
Good job. Send me a version for the master branch too when you can, please.

**Pull request merged**